src/pages/guides/using/securing_web_actions.md

```diff
@@ -26,6 +26,12 @@ If you fail in adding the authentication header or the secret is wrong, you will
 }
 ```
 
+## Non Web Action
+If your action is not a web action, you can still use your namespace credentials, base64 encoded, to call any of the actions in your namespace, as follows:
+```
+curl -X POST -H "Authorization: Basic base64-namepsace-auth" https://[your-namespaces].adobeioruntime.net/api/v1/default/my-secure-action
```

Comment:
Wondering if using `<>` around `<basic-namespace-auth>` and the above `<this-is-the-secret-hash>` is more obvious to the reader, as it may be easy to mistake being able to replace `Basic` in that line?
```
"Authorization: Basic base64-namepsace-auth" => "Authorization: Basic <base64-namepsace-auth>"
"X-Require-Whisk-Auth: this-is-the-secret-hash" => "X-Require-Whisk-Auth: <this-is-the-secret-hash>"
```
Just a thought